lib/App/perlbrew.pm

```diff
@@ -873,10 +874,39 @@ sub available_perls_with_urls {
         # if we have a $current_perl add it to the available hash of perls
         if ( $current_perl ){
             $current_perl =~ s/\.tar\.gz//;
+            push @perllist, [ $current_perl, $current_url ];
             $perls->{ $current_perl } = $current_url;
         }
     }
 
+
+    # we got impatient waiting for cpan.org to get updated to show 5.28...
+    # So, we also fetch from metacpan for anything that looks perlish,
+    # and we do our own processing to filter out the development
+    # releases and minor versions when needed (using
+    # filter_perl_available)
+    $url = 'https://fastapi.metacpan.org/v1/release/_search'
+        . '\?q=distribution:perl%20AND%20archive:perl-5.%2A.%2A.tar.gz%20AND%20NOT%20status:backpan'
+        . '\&fields=download_url\&size=100';
```

Comment:
I wonder if backslashing `&` and `?` are worng here since in single quote string `\` means a literal `\` character. However, the http response seems to be suggest that the result correct.

Interesting.

Reply:
These are backslashed because they are passed to the shell, which was trying to fork jobs into the background before I did that.  Probably the right way to fix it is to build a list of arguments and call shell() that way, rather than invoking the shell to process the arguments.